packages/lib/services/apiKey.ts

```diff
@@ -8,20 +8,20 @@ import { createHash, randomBytes } from "crypto";
 import { DatabaseError, InvalidInputError, ResourceNotFoundError } from "@formbricks/errors";
 import { cache } from "react";
 
-export const getApiKey = async (apiKey: string): Promise<TApiKey | null> => {
-  if (!apiKey) {
+export const getApiKeyFromId = async (apiKeyId: string): Promise<TApiKey | null> => {
```

Comment:
why did you rename this? I strongly prefer “getApiKey” as it’s the default method to get it by id

Reply:
also is this rename/remove of thd old function affecting existing functionality?

Reply:
renamed! also, no it does not affect existing functionality because we do not use this method anywhere else! and this method was similar to the `getApiKeyFromKey()` which still exists!